app/livechat/server/lib/Livechat.js

```diff
@@ -264,6 +264,7 @@ export const Livechat = {
 			const userData = {
 				username,
 				ts: new Date(),
+				...id && { id },
```

Comment:
Mongo's id is `_id`.

Reply:
Ooops :see_no_evil: Glad you caught this